web/index.html

```diff
@@ -104,15 +104,33 @@
     
     loadLanguageResources().finally(initialTmailApp);
 
-    _flutter.loader.load({
-      serviceWorkerSettings: {
-        serviceWorkerVersion: {{flutter_service_worker_version}},
-      },
-      onEntrypointLoaded: async function(engineInitializer) {
-        const appRunner = await engineInitializer.initializeEngine();
-        await appRunner.runApp();
-      }
-    });
+    if ('serviceWorker' in navigator) {
+      navigator
+        .serviceWorker
+        .getRegistrations()
+        .then(async function(registrations) {
+          try {
+            await Promise.all(registrations.map(function(registration) {
+              return registration.unregister();
+            }));
```

Comment:
If we do so will it cause side effects. If there are some service workers not spawned by flutter will we still destroy it?

Reply:
Yes. So far, only `flutter_service_worker` is registered. Removing all service workers also means if there is any unexpected service worker that is installed, that goes too.

Reply:
add this to `Consequences` in ADR